src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+
+  --active-state-color: #00acdc;
+}
+
+.header__content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1100px;
+  margin: 0 auto;
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:nth-last-child(1) {
```

Comment:
The selector `.nav__item:nth-last-child(1)` is correct, but it's more common to use `.nav__item:last-child` to target the last child directly. Consider using `.nav__item:last-child` for better readability and maintainability.